content/docs/Getting started/_index.md

```diff
@@ -6,3 +6,6 @@ date: 2019-02-22
 description: >
   Guides for system admins deploying OpenCue components and installing dependencies
 ---
+
+There are a lot of components in OpenCue. If you are new to OpenCue it might be useful to understand how all the components work together in the [overview](https://www.opencue.io/docs/concepts/opencue-overview/) section.
```

Comment:
I recommend updating to:

The guides in this section provide detailed instructions for installing OpenCue in a production environment. If you'd like to quickly try out OpenCue to evaluate the project for your needs, see the [Quick starts](/docs/quick-starts/).

OpenCue relies on a lot of components and project-specific terminology. If you're new to OpenCue, you might find it useful to learn how some of the components work together. For more information, see the [OpenCue overview](/docs/concepts/opencue-overview/) and [Glossary](/docs/concepts/glossary/).

Reply:
updated. please check.

Reply:
Looks good @apoorvkhare07, thank you.